src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheckTest.java

```diff
@@ -42,46 +42,46 @@ protected String getPackageLocation() {
     }
 
     @Test
-    public void testCaseSensitive()
+    public void testIllegalTokenTextTokensCheckTestCaseSensitive()
```

Comment:
testIllegalTokenText--`Tokens`--CheckTestCaseSensitive

Please remove this word from all such occurences and also update the test method names according to new input filenames that I have suggested as part of review

Reply:
ok @MANISH-K-07 got it

Reply:
@MANISH-K-07  see what is the problem now